src/components/proposals/brand/navbar/navbar.scss

```diff
@@ -0,0 +1,466 @@
+@import "../../../../assets/scss/variables";
+@import "../../../../assets/scss/utilities";
+
+.iowa-bar--narrow .logo--tab {
+  margin-right: 0px;
+}
+
+.links{
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+}
+
+.top-nav {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content:space-between
+}
+
+.top{
+  display: flex;
+  flex-direction: row;
+}
+
+.site-name-wrapper{
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  width:100%;
+  padding-left:0px;
+
+  h1 >a {
+    display: flex;
+    align-items: center;
+    margin: 0 auto 0 0;
+    text-align: left;
+    font-size: 1.5rem;
+    font-weight: 300;
+    line-height: 1;
+    font-family: "Roboto", sans-serif;
+    letter-spacing: -0.02em;
```

Comment:
Do you mind if we remove the negative `letter-spacing`?  We aren't doing that anywhere else with the brand headers.